client/src/components/admin/PageAdminProjectsUploadModal.tsx

```diff
@@ -397,35 +386,23 @@ const PageAdminProjectsUploadModalComponent: React.FC<PageAdminProjectsUploadMod
     if (state.csv.length > 0) {
       return (
         <div>
-          <Form.Group>
-            <Form.Label>Number of Expos</Form.Label>
-            <Form.Control
-              disabled={state.requesting}
-              name='numberExpos'
-              onChange={(event: any) => {
-                dispatch({ type: 'adjust-totals', numberExpos: event.target.value})
-              }}
-              type='number'
-              value={"" + state.inputNumberExpos} />
-            <Form.Text className="text-muted">
-              How many expos will there be at this event?
-            </Form.Text>
-          </Form.Group>
           {Object.values(props.tableGroups).map((tableGroup: TableGroup) => {
             return (
               <Form.Group key={tableGroup.id}>
                 <Form.Label># <strong>{tableGroup.name}</strong> Tables</Form.Label>
                 <Form.Control
                   disabled={state.requesting}
                   onChange={(event: any) => {
-                    dispatch({ type: 'adjust-counts', tableGroups: {
-                      [tableGroup.id!]: +event.target.value
-                    }});
+                    dispatch({
+                      type: 'adjust-counts', tableGroups: {
+                        [tableGroup.id!]: +event.target.value
+                      }
+                    });
                   }}
                   type='number'
                   value={"" + state.inputTableGroups[tableGroup.id!]} />
-                  <Form.Text className="text-muted">
-                    How many tables are available in the {tableGroup.name} group in each expo?

```

Comment:
do we need to change the CSV upload logic to assign all projects to round 1 instead of dividing them between the rounds (which used to be expos)?